clusterloader2/pkg/measurement/measurementmanager.go

```diff
@@ -0,0 +1,67 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package measurement
+
+import (
+	"sync"
+
+	clientset "k8s.io/client-go/kubernetes"
+)
+
+// MeasurementManager manages all measurements calls and executions.
+type MeasurementManager struct {
+	lock         sync.Mutex
+	measurements map[string]map[string]Measurement
+	clientSet    clientset.Interface
+}
+
+// CreateMeasurementManager creates new instance of MeasurementManager.
+func CreateMeasurementManager(clientSet clientset.Interface) *MeasurementManager {
+	return &MeasurementManager{
+		measurements: make(map[string]map[string]Measurement),
+		clientSet:    clientSet,
+	}
+}
+
+// Execute executes measurement based on provided identifier, metricMethod and params.
+func (mm *MeasurementManager) Execute(identifier string, metricMethod string, params map[string]interface{}) error {
+	m, err := mm.getMeasurement(identifier, metricMethod)
+	if err != nil {
+		return err
+	}
+	config := &MeasurementConfig{
+		ClientSet: mm.clientSet,
+		Params:    params,
+	}
+	return m.Execute(config)
+}
+
+func (mm *MeasurementManager) getMeasurement(identifier string, metricMethod string) (Measurement, error) {
```

Comment:
Why are you calling it metricMethod and not just method? (because it doesn't have to be a metric-related method)